### PR TITLE
Add "Do you speak prestashop", august 2023

### PR DESCRIPTION
**Pull request by l-delin**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Comment by l-delin** [comment]
Hello! My last DYSP is out 🗞
Could we get it published by Monday? @kpodemski @coffrep 🙂

ping @Julievrz

File: content/news/translation/2023/do-you-speak-prestashop-august-2023.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Do You Speak PrestaShop? – August 2023 Edition"
+subtitle: "Where you'll learn that the PrestaShop translation project has new managers"
+date: 2023-09-17
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-august-2023.md]:
```suggestion
date: 2023-09-18
```

File: content/news/translation/2023/do-you-speak-prestashop-august-2023.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Congratulations and a huge thank you to all of you, contributors! 🎉
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active.
+Indeed, some editing and rewriting may be going on, but the percentage of translation would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
+
+## Translating PrestaShop: documentation
+
+Ever had a question regarding the translation project but never knew where to look exactly?  We've taken all your comments and suggestions and put them into a specific documentation, dedicated to you, translators! 📖
+
+In it, you'll find everything about Crowdin & documentation translations, as well as the project's Terms of Use along with a few useful resources, including the PrestaShop glossary.
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-august-2023.md]:
```suggestion
In it, you'll find everything about Crowdin and documentation translations, as well as the project's Terms of Use along with a few useful resources, including the PrestaShop glossary.
```

**Review by coffrep — CHANGES_REQUESTED** [review]
Hi @l-delin a great Build article by a great person. Thanks so much for your contributions! And for the handover to two great people also @Julievrz @axelvrsc. 
I've put the date to Monday if that is okay on the article (18th)? Let me know.

**Comment by l-delin** [comment]
Hello @coffrep, seems good to me, thanks!

**Comment by Julievrz** [comment]
@kpodemski Can we publish the article today, please? Thank you in advance for your help. 🤗

**Review by coffrep — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @l-delin

**Pull request merged**